src/resources/js/admin/tickets-emails.js

```diff
@@ -182,6 +182,11 @@ tribe.tickets.emails = {};
 
 			context.arFields = arFields;
 
+			const usingTicketEmail = $document
+				.find( 'input[name=tec-tickets-emails-rsvp-use-ticket-email]' ).is( ':checked' );
```

Comment:
Can we possibly move the `tec-tickets-emails-rsvp-use-ticket-email` to the selectors object and consume it from there as we do with the others?

Reply:
I had it in the selectors first but I thought you meant to move it here with this comment: https://github.com/the-events-calendar/event-tickets/pull/2651#discussion_r1200442120

I think I understood it wrong.

Reply:
But using ticket email is a RSVP only value, so I didn't find the advantage of making it dynamic.